guide/interactions/replying-to-slash-commands.md

```diff
@@ -0,0 +1,202 @@
+    // We return if our interaction is not a slash command
+    if (!interaction.isCommand()) return; 
+	console.log(interaction);
+});
+```
+
+
+## Responding to a Command
+
+There are multiple ways of responding to a slash command, we will be covering each of these in the following segments.
+The most common way of sending a response is by using the `interaction.reply()` method:
+
+::: warning
+Initially an Interaction token is only valid for 3 Seconds, so that's the timeframe in which you are able to use the `interaction.reply()` method. We will be covering responses that require more time in the Deferred Responses section of this page.
+:::
+
+```js
+client.on('interaction', interaction => {
+    // We return if our interaction is not a Slash Command
+    if (!interaction.isCommand()) return; 
+    // Check if it is the correct command and send a response
+	if (interaction.commandName === 'ping') interaction.reply('Pong!');
+});
+```
+
+Restart your bot and then send the command to a channel your bot has access to. If all goes well, you should see something like this:
+
+<!--- vue-discord-message doesn't yet have support for inline replies/interactions/ephemeral messages -->
+<div is="discord-messages">
+	<discord-message profile="user">
+		/ping
+	</discord-message>
+	<discord-message profile="bot">
+		Pong!
+	</discord-message>
+</div>
+
+You've successfully sent a response to a slash command! This is only the beginning, there's more to look out for so let's move on to further ways of replying to a command!
+
+
+## Ephemeral Responses
+
+You may not always want everyone who has access to the channel in which a command has been used to see the response, thankfully Discord implemented a way to hide messages from everyone but the executor of the slash command. These type of messages are called `ephemeral` messages.
+
+To make a response ephemeral all you have to do is set `ephemeral: true` in the `InteractionReplyOptions` as follows:
+
+```js
+client.on('interaction', interaction => {
+    if (!interaction.isCommand()) return; 
+    // Setting the ephemeral flag to true will make the response ephemeral
+	if (interaction.commandName === 'ping') interaction.reply('Pong!', { ephemeral: true });
+});
+```
+
+Now when you run your command again, you should see something like this:
+
+<!--- vue-discord-message doesn't yet have support for inline replies/interactions/ephemeral messages -->
+<div is="discord-messages">
+	<discord-message profile="user">
+		/ping
+	</discord-message>
+	<discord-message profile="bot">
+		Pong! (ephemeral)
+	</discord-message>
+</div>
+
+That's it! You've successfully sent an ephemeral response to a slash command.
+We're not done yet, there's still more topics to cover, so let's move on to the next!
+
+
+## Editing Responses
+
+After you've send an initial response you may want to edit that response for various reasons. This can be easily achieved by making use of the `interaction.editReply()` method as seen below:
```

Comment:
```suggestion
After you've sent an initial response, you may want to edit that response for various reasons. This can be easily achieved by making use of the `interaction.editReply()` method, as seen below:
```